#NAV_MSGS
Header header
float32 ROLL
float32 PITCH
float32 YAW
float32 Uncertain_ROLL
float32 Uncertain_PITCH
float32 Uncertain_YAW
float32 WGS84_Lat
float32 WGS84_Lon
float32 WGS84_Alt
float32 NED_N
float32 NED_E
float32 NED_D
float32 VEL_N
float32 VEL_E
float32 VEL_D
float32 COMP_ACC_X
float32 COMP_ACC_Y
float32 COMP_ACC_Z
float32 COMP_GYRO_X
float32 COMP_GYRO_Y
float32 COMP_GYRO_Z
float32 UNCOMP_ACC_X
float32 UNCOMP_ACC_Y
float32 UNCOMP_ACC_Z
float32 UNCOMP_GYRO_X
float32 UNCOMP_GYRO_Y
float32 UNCOMP_GYRO_Z